modules/navier_stokes/test/tests/postprocessors/flow_rates/mass_flux_weighted_pp.i

```diff
@@ -0,0 +1,183 @@
+# Fluid properties
+mu = 1
+rho = 1
+cp = 1
+k = 1e-3
+
+# Operating conditions
+u_inlet = 1
+T_inlet = 200
+p_outlet = 10
+
+# expected temperature at outlet
+#
+# mdot * cp dT = Qdot
+# Qdot = 500
+# mdot = 1
+# cp = 1
+# dT = 500
```

Comment:
we dont quite get that, off by 2e-5 but that s because of the heat source right up to the inlet. 
If we swtich interpolation method, error goes down another factor